src/app-layout/visual-refresh/navigation.scss

```diff
@@ -17,6 +17,18 @@
   top: var(#{custom-props.$offsetTop});
   z-index: 850;
 
+  /*
+  The navigation and tools containers (that contain the toggle buttons)
+  stretch the full height of the app layout. Normally, this wouldn't be an
+  issue because they sit above the app layout's content padding.
+  
+  But if disableContentPaddings is set to true and there are buttons on the
+  left/right edges of the screen, they will be covered by the containers. So
+  we need to disable pointer events in the container and re-enable them in
+  the panels and toggle buttons.
+  */
+  pointer-events: none;
```

Comment:
Should we pull in `disableContentPaddings` to the navigation and tools components from the context? If so, we could bind the pointer events removal to the use of this property exclusively:

```
.navigation-container {
  &.disable-content-paddings {
    pointer-events: none;
  }
}

.tools-container {
  &.disable-content-paddings {
    pointer-events: none;
  }
}
```

It's not technically necessary, though it does contain the impact of the change to only these edge case circumstances.

Reply:
From @avinashbot transferring from Slack for the record:

> My worry is that this could mask any pointer-events issues we might introduce in the future. If we do it only for a small set of cases, it might be less likely that we detect it during integration or screenshot tests.

Agreed on this point. The broader the use the more likely we are to catch missing pointer events from future interactive elements in development.